fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -305,6 +309,9 @@ public long getId() {
      * @return unique id of database in string format
      */
     public String getUUID() {
+        if (CatalogMgr.isExternalCatalog(catalogName)) {
```

Comment:
why not abstact something like `ExternalDatabase()` which extends from `Database` and overrides `getUUID()`?